server/core/region.go

```diff
@@ -716,17 +716,19 @@ func (r *RegionsInfo) shouldRemoveFromSubTree(region *RegionInfo, origin *Region
 		return false
 	}
 
-	remove := false
 	if origin.leader.GetId() != region.leader.GetId() {
-		remove = true
-	} else if checkPeersChange(origin.GetVoters(), region.GetVoters()) {
-		remove = true
-	} else if checkPeersChange(origin.GetLearners(), region.GetLearners()) {
-		remove = true
-	} else if checkPeersChange(origin.GetPendingPeers(), region.GetPendingPeers()) {
-		remove = true
-	}
-	return remove
+		return true
```

Comment:
```suggestion
		return origin.leader.GetId() != region.leader.GetId() || checkPeersChange(origin.GetVoters(), region.GetVoters()) || checkPeersChange(origin.GetLearners(), region.GetLearners())||checkPeersChange(origin.GetPendingPeers(), region.GetPendingPeers()) 
```

Reply:
I meant something  below

Reply:
I think the present way of writing if separately is clearer and easier to understand. What do you think?^_^